### PR TITLE
Exercise 13

### PR DESCRIPTION
**Pull request by vinhcotam**
## Related Tickets
- [#Task xxxxx: Content](https://edu-redmine.sun-asterisk.vn/issues/xxxx)
## WHAT
- Notification excercise
## Evidence (Screenshot or Video)
image
image

## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/blob/master/eng/android/coding_convention_android_kotlin.md | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(Other notes)*

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -120,4 +130,5 @@
                 android:value="" />
         </activity>
     </application>
-</manifest>
+
+</manifest>
```
Comment (hungld-1302) [inline app/src/main/AndroidManifest.xml]: endline

Comment (ghost) [inline app/src/main/AndroidManifest.xml]: endline

File: app/src/main/java/com/sun/android/ex_slide20/NotificationActivity.kt [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+import androidx.core.app.NotificationCompat
+import com.sun.android.R
+import com.sun.android.databinding.ActivityNotificationBinding
+
+
+class NotificationActivity : AppCompatActivity() {
+    private val binding by lazy {
+        ActivityNotificationBinding.inflate(layoutInflater)
+    }
+    private lateinit var mNotifyManager: NotificationManager
```
Comment (ghost) [inline app/src/main/java/com/sun/android/ex_slide20/NotificationActivity.kt]: không dùng lateinit

File: app/src/main/java/com/sun/android/ex_slide20/NotificationActivity.kt [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        ActivityNotificationBinding.inflate(layoutInflater)
+    }
+    private var mNotifyManager: NotificationManager? =null
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(binding.root)
+        createNotificationChannel()
+        binding.buttonNotifyMe.setOnClickListener() {
+            sendNotification()
+        }
```
Comment (minhnn-1279) [inline app/src/main/java/com/sun/android/ex_slide20/NotificationActivity.kt]:
```suggestion
        binding.buttonNotifyMe.setOnClickListener{
            sendNotification()
        }
```

**Pull request merged**